src/Paginator.php

```diff
@@ -158,6 +158,37 @@ public function unseekable()
     }
 
     /**
+     * Define options from an associative array.
+     *
+     * @param  (bool|int|string[])[]
```

Comment:
@chitoku-k Is this annotation valid?

Reply:
@mpyw Scrutinizer did not emit errors, maybe.